docs: Exception tab (#585)
grafana/traces-drilldown: docs/sources/investigate/analyze-tracing-data.md

![Duration metric showing root cause latency](/media/docs/explore-traces/traces-drilldown-traces-view-v1.2.png)

> ## Use the Exceptions tab (Errors metric only)

Author: ```suggestion
## Use the Exceptions tab 
```

Use the **Exceptions** tab to see which exception messages are occurring within your current filters and time range. Exceptions are grouped by message so you can identify the most frequent failures and access the affected traces.

Examples:
- Inspect individual exceptions by clicking a message to open the **Trace list** pre-filtered for that message, so you can inspect individual traces immediately.
- Narrow exceptions by service, environment, namespace, or any span/resource attribute by combining with the **Filters** bar.

![Exceptions tab](/media/docs/explore-traces/traces-drilldown-error-exceptions-v1.2.png)

The Exceptions tab has a table with the following columns:
- **Message**: Exception text grouped by unique message
- **Type**: Exception class or code when available
- **Trace service**: Service emitting the exception most recently
- **Occurrences**: Count of matching exceptions for the selected range

> - **Time series**: Sparkline of occurrences over time

Author: ```suggestion
- **Frequency**: Sparkline of occurrences over time
```

- **Last seen**: The most recent occurrence

The **Exceptions** tab is available when the **Errors** metric is selected and respects the current time range and filters.

> It works with both **Root spans** and **All spans** selections. The results reflect the data in scope.

Reviewer: Technically it can work with any primary metric if there are exceptions.

Author: I'll update the section. I documented as Exceptions only working with Errors because it never showed upf for Rate or Duration.

> It works with both **Root spans** and **All spans** selections. The results reflect the data in scope.

Author: ```suggestion
The **Exceptions** tab is available for primary metric if there are exceptions. 
The tab respects the current time range and filters.
For example, when Errors are selected, **Exceptions** works with both **Root spans** and **All spans** selections. The results reflect the data in scope.
```
